RELEASE_NOTES.md

```diff
@@ -0,0 +1,53 @@
+# Cosmos SDK v0.43.0 Release Notes
+
+This release introduces several new important updates to the Cosmos SDK. The release notes below provide an overview of the larger high-level changes introduced in the v0.43 release series.
+
+That being said, this release does contain many more minor and module-level changes besides those mentioned below. For a comprehsive list of all breaking changes and improvements since the v0.42 "Stargate" release series, please see the
+[CHANGELOG](https://github.com/cosmos/cosmos-sdk/blob/v0.43.0-rc0/CHANGELOG.md).
+
+## Two new modules: `x/authz` and `x/feegrant`
+
+The v0.43 release focused on simplifying [keys and fee management](https://github.com/cosmos/cosmos-sdk/issues/7074) for SDK users, by introducing the two following modules:
```

Comment:
Maybe we should add a note that both modules have a new structure and we believe it's more clear and good to follow.

Reply:
added